### PR TITLE
Fixes templating parsing for manually created vars

### PR DESCRIPTION
**Pull request by mothershipper**
#### What this PR does

When you create a dashboard variable from within the Grafana UI, it creates the query field as an object. The object contains a refId as well as the original query.

The existing code expects the query to be a string, not an object and will therefor miss any metrics used if the variable was created in the UI.

#### Which issue(s) this PR fixes or relates to

Not sure if there is an issue filed, but we viewed our dashboard JSON and our dashboard variables look something like this:

```
  {
  "query": {
     "query": "label_values(kube_service_info{cluster=\"$cluster\"})",
     "refId": "StandardVariableQuery"
   },
   "type": "query"
}
```

The code being changed expects the JSON to look like:

```
{
  "query": "label_values(kube_service_info{cluster=\"$cluster\"})",
   "type": "query"
}
```

Change should allow the analyze command to handle both formats.  I'm guessing the latter is often the case for dashboards maintained by hand, while our variables were created via the Grafana UI.

#### Checklist

- [x] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: pkg/mimirtool/commands/testdata/apiserver.json [hunk 1]
```diff
@@ -1461,11 +1461,34 @@
...
+            "name": "manuallycreated",
+            "options": [ ],
+            "query": {
+                "query": "label_values(kube_pod_info, cluster)",
+                "refId": "ManuallyCreatedVar"
+            },
+            "refresh": 2,
+            "regex": "",
+            "sort": 1,
+            "tagvaluesquery": "",
```
Comment (dimitarvdimitrov) [inline pkg/mimirtool/commands/testdata/apiserver.json:1480]: I ran the tests with camel-cased and lowercase JSON and the detection works in both times.

I checked this on Grafana 9.1 and 9.4 and the JSON model seems to be camel-cased. Where did you get the lowercase one?

**Review by dimitarvdimitrov — APPROVED** [review]
Thanks for fixing this! I have a question on the casing in the test JSON, but otherwise LGTM

File: pkg/mimirtool/commands/testdata/apiserver.json (hunk 1 above)
Comment (mothershipper) [reply pkg/mimirtool/commands/testdata/apiserver.json:1480]: I copied the JSON block that was already in this testdata file and just modified the query field 😬

Comment (mothershipper) [reply pkg/mimirtool/commands/testdata/apiserver.json:1480]: I'm not sure what your merge flow looks like, but I could either fix just the added chunk in this PR and amend the commit and fix the other as a separate commit within this PR (assuming rebase+merge)

**Review by dimitarvdimitrov — APPROVED** [review]
LGTM, thank you!

**Comment by dimitarvdimitrov** [comment]
I'll wait for others to also take a look before merging

File: pkg/mimirtool/commands/testdata/apiserver.json (hunk 1 above)
Comment (mothershipper) [reply pkg/mimirtool/commands/testdata/apiserver.json]: my bad, must not be getting enough sleep 🙈  assuming my editor went rouge and I didn't notice.

Just pushed a fix, but let me squash that down

**Review by pracucci — APPROVED** [review]
LGTM, thanks!

**Pull request merged**